From bae0eed36b4f7cab4f21039c81ca01be9f933445 Mon Sep 17 00:00:00 2001
From: Julien Massot <jmassot@aldebaran-robotics.com>
Date: Thu, 21 Jul 2011 11:46:39 +0000
Subject: [PATCH 2/2] gsupplicant: set the cipher even if security is WEP

This patch fix an issue with WEP40, i tested it with WEP40 WEP104 and open AP.

It set the group cipher even if security is WEP otherwise connman won't connect to WEP40 service.
---
 gsupplicant/supplicant.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/gsupplicant/supplicant.c b/gsupplicant/supplicant.c
index bbb04ae..e5743f0 100644
--- a/gsupplicant/supplicant.c
+++ b/gsupplicant/supplicant.c
@@ -2817,6 +2817,7 @@ static void add_network_security(DBusMessageIter *dict, GSupplicantSSID *ssid)
 	case G_SUPPLICANT_SECURITY_WEP:
 		key_mgmt = "NONE";
 		add_network_security_wep(dict, ssid);
+		add_network_security_ciphers(dict, ssid);
 		break;
 	case G_SUPPLICANT_SECURITY_PSK:
 		key_mgmt = "WPA-PSK";
-- 
1.7.6

